css/style.css

```diff
@@ -0,0 +1,485 @@
+}
+
+.table-2 {
+  display: flex;
+  justify-content: center;
+  width: 600px;
+  height: 95%;
+  flex-wrap: wrap;
+}
+
+
+
+
+.card-wrapper {
+  outline: 2px solid blue;
+  /* width: 31%; */
+  width: 218px;
+  height: 341px;
+  text-align: center;
+  perspective: 1200;
+  -webkit-perspective: 1200;
+  -moz-transform: perspective(1200px);
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d; 
+}
+
+.card-inner {
+  outline: 2px solid red;
+  position: relative;
+  width: 100%;
+  transform-style: preserve-3d;
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  transition: .7s linear;  
+  -webkit-transition: .7s linear;
+  -moz-transition: .7s linear;
+  -o-transition: .7s linear;
+}
+
+.front-side, .back-side {
+  background-size: contain;
+  background-repeat: no-repeat;
+  outline: 2px solid yellow;
+  width: 100%;
+  /* width: 218px; */
+  height: 341px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  -webkit-backface-visibility: hidden;
+  -moz-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.back-side {
+  /* width: 100%; */
+  background-image: url(../img/back-side.png);
+  -webkit-transform: rotateY(0deg);
+  -moz-transform: rotateY(0deg);
+  -ms-transform: rotateY(0deg);
+  transform: rotateY(0deg);
+}
+
+.front-side {
+  background-image: url(../img/game-over.png);
+  /* background-size: contain; */
+  /* background-repeat: no-repeat; */
+  -webkit-transform: rotateY(-180deg);
+  -moz-transform: rotateY(-180deg);
+  -ms-transform: rotateY(-180deg);
+  transform: rotateY(-180deg);
+}
+
+.front-side--win {
+  background-image: url(../img/winner.png)
+}
+
+.flip {
+  -webkit-transform: rotateY(180deg);
+  -moz-transform: rotateY(180deg);
+  -ms-transform: rotateY(180deg);
+  transform: rotateY(180deg);
+}
+
+.table-2 .card-wrapper {
+  width: 26%;
+}
+
+.table-2 .card-inner {
+  width: 99%;
+}
+
+.table-2 .card, .table-2 .result {
+  width: 93%;
+}
+
+@keyframes rotation {
+  from {transform: scale(1);}
+  
+  50% {
+    transform: rotate(-3.08deg);
+    transform: scale(1.05);
+  }
+  
+  to {
+    transform: scale(1.1);
+    transform: rotate(-3.08deg);
+  }
+}
+
+.card:hover {
+  animation: .7s rotation both;
+}
+
+@media (max-width: 1200px) {
+
+  button {
+    margin-top: 40px;
+  }
+  
+}
+
+@media (max-width: 980px) {
+
+/*   .container {
+    max-width: 92%;
+  } */
+
+  .first-card, .second-card {
+    width: 160px;
+  }
+
+  .first-card {
+    left: 55%;
+  }
+
+  .second-card {
+    left: 65%;
+  }
+
+  .level:hover::before {
+    /* display: none; */
+  }
+
+}
+
+@media (max-width: 768px) {
+
+  .table-0 {
+    width: 580px;
+  }
+
+  .table-1 {
+    /* height: 95%; */
+  }
+
+  .table-2 {
+    /* height: 97%; */
+  }
+
+  .card-wrapper {
+    /* width: 31%; */
+    width: 180px;
+    height: 282px;
+  }
+
+  .front-side, .back-side {
+    /* width: 100%; */
+    width: 180px;
+    height: 282px;
+  }  
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)